server/boot/story.js

```diff
@@ -104,6 +105,23 @@ module.exports = function(app) {
     );
   }
 
+  function userStories({ body: { search = '' } = {} }, res, next) {
+    return app.dataSources.db.connector
+      .collection('story')
+      .find({
+        'author.username': search
+      })
```

Comment:
It is prevented by https://github.com/bugron/FreeCodeCamp/blob/feature/find-posted-news-by-username/server/views/stories/news-nav.jade#L36, isn't it?